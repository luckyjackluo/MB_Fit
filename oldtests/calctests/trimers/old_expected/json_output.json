{
    "molecule": "O   8.64297000000000e-01 -1.71381800000000e+00 -4.73367000000000e-01\nH   1.68527300000000e+00 -1.20159100000000e+00 -3.01982000000000e-01\nH   1.14997000000000e+00 -2.58689700000000e+00 -7.56299000000000e-01\nO  -7.30697000000000e-01  4.05669000000000e-01 -1.63668400000000e+00\nH  -2.64780000000000e-01 -4.35767000000000e-01 -1.50730100000000e+00\nH  -1.61069500000000e+00  2.46805000000000e-01 -1.24492900000000e+00\nO  -7.02299000000000e-01 -4.94721000000000e-01  1.65945100000000e+00\nH  -2.01238000000000e-01 -1.09653500000000e+00  1.08595400000000e+00\nH  -2.82609000000000e-01  3.63229000000000e-01  1.49278700000000e+00",
    "frag_energies": {
        "E(0, 1, 2)": -224.90664376436285,
        "E(0, 1)": -149.93586907477547,
        "E(0, 2)": -149.93470912363466,
        "E(1, 2)": -149.93022709382893,
        "E(0,)": -74.96364694161441,
        "E(1,)": -74.96490292927481,
        "E(2,)": -74.96489418272135
    },
    "n-body_energies": {
        "V_1B": -224.8934440536106,
        "V_2B": -0.01391718501787409,
        "V_3B": 0.0007174742656275157
    },
    "k-body_energies": {
        "K(0, 1, 2)_3B": 0.0007174742656275157,
        "K(0, 1)_2B": -0.007319203886225978,
        "K(0, 2)_2B": -0.0061679992988956656,
        "K(1, 2)_2B": -0.000429981832780868,
        "K(0,)_1B": -74.96364694161441,
        "K(1,)_1B": -74.96490292927481,
        "K(2,)_1B": -74.96489418272135
    }
}