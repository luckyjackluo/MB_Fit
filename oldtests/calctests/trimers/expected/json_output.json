{
    "ID": "63b562bcedc217f57e7a47439e65a7a97274f33f",
    "molecule": "O    8.64297000000000e-01  -1.71381800000000e+00  -4.73367000000000e-01\nH    1.68527300000000e+00  -1.20159100000000e+00  -3.01982000000000e-01\nH    1.14997000000000e+00  -2.58689700000000e+00  -7.56299000000000e-01\nO   -7.30697000000000e-01   4.05669000000000e-01  -1.63668400000000e+00\nH   -2.64780000000000e-01  -4.35767000000000e-01  -1.50730100000000e+00\nH   -1.61069500000000e+00   2.46805000000000e-01  -1.24492900000000e+00\nO   -7.02299000000000e-01  -4.94721000000000e-01   1.65945100000000e+00\nH   -2.01238000000000e-01  -1.09653500000000e+00   1.08595400000000e+00\nH   -2.82609000000000e-01   3.63229000000000e-01   1.49278700000000e+00",
    "model": "HF/STO-3G",
    "frag_energies": {
        "E(0,)": -74.96364694162102,
        "E(1,)": -74.96490292923929,
        "E(2,)": -74.96489418270039,
        "E(0, 1)": -149.93586907469853,
        "E(0, 2)": -149.9347091238848,
        "E(1, 2)": -149.9302270939695,
        "E(0, 1, 2)": -224.90664376421483
    },
    "n-body_energies": {
        "V_1B": -224.8934440535607,
        "V_2B": -0.013917185431409962,
        "V_3B": 0.0007174747772751289
    },
    "individual_n-body_energies": {
        "V(0,)_1B": -74.96364694162102,
        "V(1,)_1B": -74.96490292923929,
        "V(2,)_1B": -74.96489418270039,
        "V(0, 1)_2B": -0.007319203838221711,
        "V(0, 2)_2B": -0.0061679995633880935,
        "V(1, 2)_2B": -0.00042998202982857947,
        "V(0, 1, 2)_3B": 0.0007174747772751289
    }
}